packages/common/src/SVGWidget.ts

```diff
@@ -225,6 +227,32 @@ export class SVGWidget extends Widget {
                     .style("top", 0)
                     .style("left", 0)
                     ;
+                const linearGradient = this._defs.append("linearGradient")
+                    .attr("id", `${this.id()}_grad`)
+                    .attr("y1", "1")
+                    .attr("y2", "0")
+                    .attr("x2", "0")
+                    ;
+                linearGradient.append("stop")
+                    .attr("offset", "0.1")
+                    .attr("stop-color", "white")
+                    .attr("stop-opacity", "0")
+                    ;
+                linearGradient.append("stop")
+                    .attr("offset", "1")
+                    .attr("stop-color", "white")
+                    .attr("stop-opacity", "1")
+                    ;
+                this._defs.append("mask")
+                    .attr("id", `${this.id()}_fade`)
+                    .attr("maskContentUnits", "userSpaceOnUse")
+                    .append("rect")
+                    .attr("x", "-150")
+                    .attr("y", "-150")
+                    .attr("width", "300")
+                    .attr("height", "300")
+                    .attr("fill", `url(#${this.id()}_grad)`)
+                    ;
```

Comment:
Should this be in _all_ SVGWidget items?  Why not just add it specific to HexBins?

Reply:
I was thinking of it as a global capability on par with `_glow`. It could be a pretty neat feature for Bar/Bubble/Area etc

Reply:
In which case add it to the Column chart as well?